packages/mongo/oplog_tailing.js

```diff
@@ -121,15 +121,51 @@ _.extend(OplogHandle.prototype, {
   // currently visible.
   // XXX become convinced that this is actually safe even if oplogConnection
   // is some kind of pool
-  waitUntilCaughtUp: function () {
+  _oplogCheckRunning: false,
+  _oplogCheckCallbacks: [],
```

Comment:
Doesn't this create a single `OplogHandle.prototype._oplogCheckCallbacks` object instead of one per OplogHandle? These fields should be initialized in the constructor.